examples/webgl_renderer_pathtracer.html

```diff
@@ -0,0 +1,496 @@
+
+				const aspect = w / h;
+				camera.aspect = aspect;
+				camera.updateProjectionMatrix();
+
+			}
+
+			function createGUI() {
+
+				if ( gui ) {
+
+					gui.destroy();
+
+				}
+
+				gui = new GUI();
+				gui.add( params, 'enable' );
+				gui.add( params, 'pause' );
+				gui.add( params, 'toneMapping' );
+				gui.add( params, 'transparentBackground' ).onChange( v => {
+
+					pathTracer.material.backgroundAlpha = v ? 0 : 1;
+					renderer.setClearAlpha( v ? 0 : 1 );
+					pathTracer.reset();
+
+				} );
+				gui.add( params, 'resolutionScale', 0.1, 1.0 ).onChange( onWindowResize );
+				gui.add( params, 'tiles', 1, 3, 1 ).onChange( v => {
+
+					pathTracer.tiles.set( v, v );
+
+				} );
+				gui.add( params, 'roughness', 0, 1 ).name( 'floor roughness' ).onChange( () => {
+
+					pathTracer.reset();
+
+				} );
+				gui.add( params, 'metalness', 0, 1 ).name( 'floor metalness' ).onChange( () => {
+
+					pathTracer.reset();
+
+				} );
+				gui.add( params, 'download' ).name( 'download image' );
+
+
+			}
+
+			//
+
+			function render() {
+
+				requestAnimationFrame( render );
+
+				if ( ! sceneInfo ) {
+
+					return;
+
+				}
+
+				renderer.toneMapping = params.toneMapping ? THREE.ACESFilmicToneMapping : THREE.NoToneMapping;
+
+				if ( pathTracer.samples < 1.0 || ! params.enable ) {
+
+					renderer.render( scene, camera );
+
+				}
+
+				if ( params.enable && delaySamples === 0 ) {
+
+					const samples = Math.floor( pathTracer.samples );
+					samplesEl.innerText = `samples: ${ samples }`;
+
+					floor.material.roughness = params.roughness;
+					floor.material.metalness = params.metalness;
+
+					pathTracer.material.materials.updateFrom( sceneInfo.materials, sceneInfo.textures );
+					pathTracer.material.filterGlossyFactor = 0.5;
+					pathTracer.material.physicalCamera.updateFrom( camera );
+
+					camera.updateMatrixWorld();
+
+					if ( ! params.pause || pathTracer.samples < 1 ) {
+
+						pathTracer.update();
+
+					}
+
+					renderer.autoClear = false;
+					fsQuad.render( renderer );
+					renderer.autoClear = true;
+
+				} else if ( delaySamples > 0 ) {
+
+					delaySamples --;
+
+				}
+
+				samplesEl.innerText = `samples: ${ Math.floor( pathTracer.samples ) }`;
+
+			}
+
+			function onProgress( xhr ) {
+
+				if ( xhr.lengthComputable ) {
+
+					updateProgressBar( xhr.loaded / xhr.total );
+
+					console.log( Math.round( xhr.loaded / xhr.total * 100, 2 ) + '% downloaded' );
+
+				}
+
+			}
+
+			function onError( error ) {
+
+				const message = 'Error loading model';
+				progressBarDiv.innerText = message;
+				console.log( message );
+				console.error( error );
+
+			}
+
+			function showProgressBar() {
+
+				document.body.appendChild( progressBarDiv );
+
+			}
+
+			function hideProgressBar() {
+
+				document.body.removeChild( progressBarDiv );
+
+			}
+
+			function updateProgressBar( fraction ) {
+
+				progressBarDiv.innerText = 'Loading... ' + Math.round( fraction * 100, 2 ) + '%';
```

Comment:
The progress bar shows "599%" at some point.

Reply:
This may be due to githack or a the localhost server depending on where you're seeing it. I've seen the same when testing the LDrawLoader example, as well.